scala-package/core/pom.xml

```diff
@@ -114,5 +114,10 @@
       <groupId>org.scalacheck</groupId>
       <artifactId>scalacheck_${scala.binary.version}</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.scala-lang</groupId>
+      <artifactId>scala-parser-combinators</artifactId>
+      <version>2.11.0-M4</version>
```

Comment:
I wonder whether we really need this package. The less dependencies, the better.

If so, pls put the version into root `pom.xml`, and use `scala.binary.version` to make it work for 2.10 as well.
pls refer to https://github.com/scala/scala-module-dependency-sample